best_practices/security.rst

```diff
@@ -167,7 +166,7 @@ need to repeat the expression code using Twig syntax:
         <a href=""> ... </a>
     {% endif %}
 
-The easiest solution - if your logic is simple enough - is to add a new method
+The best solution - if your logic is simple enough - is to add a new method
```

Comment:
not sure if its the "best" solution

Reply:
`A better` or `An easier`? - It's not the "easiest" but much great than using an expression.

Reply:
`A better` 👍